community/index/src/test/java/org/neo4j/index/internal/gbptree/FormatCompatibilityTest.java

```diff
@@ -97,8 +94,8 @@ public void shouldDetectFormatChange() throws Throwable
         // WHEN reading from the tree
         // THEN everything should work, otherwise there has likely been a format change
         PageCache pageCache = pageCacheRule.getPageCache( fsRule.get() );
-        try ( GBPTree<MutableLong,MutableLong> tree = new GBPTree<>( pageCache, storeFile, new SimpleLongLayout(), 0,
-                NO_MONITOR, NO_HEADER_READER, NO_HEADER_WRITER ) )
+        try ( GBPTree<MutableLong,MutableLong> tree =
+                new GBPTreeBuilder( pageCache, storeFile, new SimpleLongLayout() ).build() )
```

Comment:
`GBPTreeBuilder( ... )` is unchecked. Should be `GBPTreeBuilder<>( ... )`. My search turned up with 13 occurrences of this, the same comment applies to all of them.